proxyd/integration_tests/consensus_custom_config_test.go

```diff
@@ -129,3 +130,159 @@ func TestConsensusSkipSyncTest(t *testing.T) {
 		require.Equal(t, 2, len(consensusGroup))
 	})
 }
+
```

Comment:
Small thing:

Can we have something like 
```go
	initSetupAndAssetions := func() {
		reset()
		update()
		
		require.Equal(t, "0x101", bg.Consensus.GetLatestBlockNumber().String())
		require.Equal(t, "0xe1", bg.Consensus.GetSafeBlockNumber().String())
		require.Equal(t, "0xc1", bg.Consensus.GetFinalizedBlockNumber().String())
	}
```
and use this as the first line under every t.Run()?

(improves the readability, ensures that every subtest starts from a fresh slate independent of the result of the previous sub-test)